contracts/MarketOracle.sol

```diff
@@ -54,7 +54,7 @@ contract MarketOracle is IOracle {
     }
 
     modifier onlyMarketOwner(
-        bytes32 messageHash,
+        bytes16 messageHash,
```

Comment:
There's no reason for this to be a bytes16, instead of bytes32

Reply:
Good catch, this is a mistake.